openpower/configs/zz_defconfig

```diff
@@ -46,6 +46,7 @@ BR2_TARGET_ROOTFS_CPIO_XZ=y
 BR2_OPENPOWER_PLATFORM=y
 BR2_OPENPOWER_POWER9=y
 # BR2_PACKAGE_PPE42_TOOLCHAIN is not set
+BR2_PACKAGE_ALTERNATE_GCC=y
```

Comment:
Are we going to set this variable for P10 configs too?

Reply:
we could if necessary... I believe we should strive to use buildroot's gcc + binutils the best we can, but if for some reason we need to update buildroot (say to fix security holes or bugs that are only fixed in more modern versions), we could set this setting, as well as the ALTERNATE_*_VERSION to keep hostboot and occ behind if necessary